GliaWidgets/Sources/Coordinators/EngagementCoordinator/EngagementCoordinator.swift

```diff
@@ -1,12 +1,33 @@
 import UIKit
 import Foundation
 
+extension EngagementCoordinator {
+    enum EngagementLaunching {
```

Comment:
I think it is worth documenting this enum for clarity.

Reply:
Added 67482f9aa893c5192af9423c2cfa1e2b107c02d1